armadillo_cli.py

```diff
@@ -0,0 +1,20 @@
+import typer
+import os
+
+import utils
+import armadillo_cli_add
+import armadillo_cli_list
+import armadillo_cli_delete
+import armadillo_cli_reload
+
+if not os.path.exists(utils.armadillo_dir_path):
+    os.mkdir(utils.armadillo_dir_path)
+
+app = typer.Typer()
+app.add_typer(armadillo_cli_add.app, name="add", help="Add new policy      (policy|policy-dns|policy-dpi)")
+app.add_typer(armadillo_cli_list.app, name="list", help="List all policies   (policy|policy-dns|policy-dpi)")
```

Comment:
```suggestion
app.add_typer(armadillo_cli_list.app, name="list", help="List all policies (policy|policy-dns|policy-dpi)")
```

Reply:
For keep all the commands' args in the same line